sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/FileQueryInputStream.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+import com.azure.storage.common.implementation.FluxInputStream;
+import reactor.core.publisher.Flux;
+
+import java.nio.ByteBuffer;
+
+/**
+ * An <code>InputStream</code> interface that represents the stream to use for reading the query response.
+ */
+public class FileQueryInputStream extends FluxInputStream {
```

Comment:
Same feedback here. I think APIs should just return InputStream. We don't need this type.